### PR TITLE
EZEE-3003: Database update script

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/EZEE-3003
| Versions      | all

Note: merge after relevant releases are out.

### CODE REVIEW

File: docs/updating/4_update_2.5.md [hunk 1]
```diff
@@ -46,6 +46,14 @@ Additionally, reindex the content:
 php bin/console ezplatform:reindex
 ```
 
+### v2.5.18
+
+To update to v2.5.18, additionally run the following update script:
+
+``` sql
+ALTER TABLE ezpage_attributes MODIFY value LONGTEXT;
```
Comment (webhdx) [inline docs/updating/4_update_2.5.md:54]: It should probably clearly state that query is only meant for MySQL.

File: docs/updating/4_update_3.2.md [hunk 2]
```diff
@@ -30,6 +30,14 @@ If you are updating from a version prior to 3.0, you have to implement all the c
 
     `psql <database_name> < upgrade/db/postgresql/ezplatform-3.1.0-to-3.2.0.sql`
 
+### v3.2.6
+
+To update to v3.2.6, additionally run the following update script:
+
+``` sql
+ALTER TABLE ezpage_attributes MODIFY value LONGTEXT;
```
Comment (webhdx) [inline docs/updating/4_update_3.2.md:38]: In 3.2 you should run queries from the *.sql file: `upgrade/db/mysql/ezplatform-3.2.5-to-3.2.6.sql`

File: docs/updating/updating_to_3.3.md [hunk 3]
```diff
@@ -163,6 +163,14 @@ Check the Location ID of the "Components" Content item and set it as a value of
 If you are upgrading between Ibexa Commerce versions,
 add the `content/read` Policy with the Owner Limitation set to `self` to the "Ecommerce registered users" Role.
 
+### v3.3.2
+
+To update to v3.3.2, additionally run the following update script:
+
+``` sql
+ALTER TABLE ezpage_attributes MODIFY value LONGTEXT;
```
Comment (webhdx) [inline docs/updating/updating_to_3.3.md:171]: In 3.3 we moved upgrade scripts to `vendor/ibexa/installer/upgrade/db/` so developers should run scripts like in 3.2:
`mysql -u <username> -p <password> <database_name> < vendor/ibexa/installer/upgrade/db/mysql/ibexa-3.3.1-to-3.3.2.sql`

**Review by webhdx — CHANGES_REQUESTED** [review]

**Review by mnocon — APPROVED** [review]

File: docs/updating/4_update_2.5.md [hunk 4]
```diff
@@ -46,6 +46,14 @@ Additionally, reindex the content:
 php bin/console ezplatform:reindex
 ```
 
+### v2.5.18
+
+To update to v2.5.18, additionally run the following update script, if you are using MySQL:
```
Comment (dabrt) [inline docs/updating/4_update_2.5.md:51]:
```suggestion
To update to v2.5.18, if you are using MySQL, additionally run the following update script:
```

File: docs/updating/4_update_3.2.md [hunk 5]
```diff
@@ -30,6 +30,14 @@ If you are updating from a version prior to 3.0, you have to implement all the c
 
     `psql <database_name> < upgrade/db/postgresql/ezplatform-3.1.0-to-3.2.0.sql`
 
+### v3.2.6
+
+To update to v3.2.6, additionally run the following update script, if you are using MySQL:
```
Comment (dabrt) [inline docs/updating/4_update_3.2.md:35]:
```suggestion
To update to v3.2.6, if you are using MySQL, additionally run the following update script:
```

File: docs/updating/updating_to_3.3.md [hunk 6]
```diff
@@ -163,6 +163,14 @@ Check the Location ID of the "Components" Content item and set it as a value of
 If you are upgrading between Ibexa Commerce versions,
 add the `content/read` Policy with the Owner Limitation set to `self` to the "Ecommerce registered users" Role.
 
+### v3.3.2
+
+To update to v3.3.2, additionally run the following update script, if you are using MySQL:
```
Comment (dabrt) [inline docs/updating/updating_to_3.3.md:168]:
```suggestion
To update to v3.3.2, if you are using MySQL, additionally run the following update script:
```

**Review by dabrt — APPROVED** [review]

**Review by webhdx — APPROVED** [review]

**Pull request merged**